src/components/spreadsheet/table-wrapper.tsx

```diff
@@ -423,6 +424,19 @@ export const TableWrapper: FunctionComponent<TableWrapperProps> = ({
         setColumnData(generateTableColumns());
     }, [generateTableColumns]);
 
+    const { modificationDialog, handleOpenModificationDialog } = useEquipmentModification({
+        studyUuid,
```

Comment:
```suggestion
        studyUuid,
        currentRootNetworkUuid,
```

we can pass currentRootNetworkUuid here